configs/components/facter-source-gem.rb

```diff
@@ -12,8 +12,13 @@
   pkg.install_file('Makefile.erb', "#{settings[:gemdir]}/ext/facter")
 
   pkg.configure do
+    if platform.is_windows?
```

Comment:
Normally we keep cases like this outside the 'configure'/'install' steps. Was there any reason to keep it inside `pkg.configure` ?

Reply:
Fair enough. I was trying to keep it to the minimal scope, but I'll match the rest of the style used

Reply:
Updated